tools/smxtools/smxdasm/File.cs

```diff
@@ -131,6 +135,52 @@ public SmxFile(BinaryReader br)
                 }
             }
             UnknownSections = unknown.ToArray();
+
+            // Disassemble all functions right away to find all called functions.
+            if (DebugSymbols != null)
+            {
+                foreach (var entry in DebugSymbols.Entries)
+                {
+                    if (entry.Ident != SymKind.Function)
+                        continue;
+                    try
```

Comment:
Rather than duplicate this try block 3 times, could you have a helper function like "TryDisassemble"?